h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -73,7 +73,7 @@
 
     // H2O GBM options
     public boolean _quiet_mode = true;
-    public MissingValuesHandling _missing_values_handling;
+    public MissingValuesHandling _missing_values_handling = MissingValuesHandling.MeanImputation;
```

Comment:
XGBoost doesn't actually expose this option to users

https://github.com/h2oai/h2o-3/blob/master/h2o-extensions/xgboost/src/main/java/hex/schemas/XGBoostV3.java#L169

IMHO this should be removed from the code base completely because it doesn't make sense in case of XGBoost - probably copied from GLM because of DataInfo re-use.

@honzasterba or @Pscheidl - can you guys fix this in H2O?

Reply:
will do https://0xdata.atlassian.net/browse/PUBDEV-6724